samples/nrf9160/nrf_cloud_rest_device_message/Kconfig

```diff
@@ -29,6 +29,10 @@ config REST_DEVICE_MESSAGE_DO_JITP
 	  nRF Cloud, enable this option and press the configured button when
 	  prompted after the sample starts.
 
+config REST_DEVICE_MESSAGE_USE_RTT
```

Comment:
now this config isn't used... it could be removed.
the overlay is all you need.

Reply:
It will be used later when I get back at this task.

Reply:
What are you planning to add that requires the flag? This PR seems complete to me, unless I'm missing something

Reply:
seems like you could just use `CONFIG_RTT_CONSOLE` to determine if RTT was in use?

Reply:
I think that's what I need, will remove the `USE_RTT` config